src/UI/templates/default/Deck/deck.less

```diff
@@ -21,3 +21,23 @@
     }
   }
 }
+
+@media (min-width: @screen-sm-min) and (max-width: @screen-sm-max) {
+  .row-eq-height {
+
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    flex: 3;
+    flex-wrap: wrap;
+    
+    > div {
+      display: -webkit-box;
+      display: -webkit-flex;
+      display: -ms-flexbox;
```

Comment:
Why is this only needed here, an not also in the above @screen-md-min section (which then again could be merged with this section here).